exercise/cache/points.py

```diff
@@ -246,8 +248,8 @@ def get_max_deviations(
                             )
                         )
                     )
-                    .values('exercise')
-                    .annotate(max_deviation=Max(field))
+                    .values('exercise', *extra_fields)
```

Comment:
This row might be buggy. It's supposed to merely include some extra fields from the maximum deviation (such as "without_late_penalty"), but I think the fields will actually affect the grouping here. Now the queryset might return multiple rows per exercise, if there are multiple deviations from different sources (one from the user and one from one of the user's group members).